src/components/GoBackHome.js

```diff
@@ -0,0 +1,14 @@
+import { useHistory } from "react-router-dom";
+import "./GoBackHome.css";
+export default function GoBackHome({ text }) {
```

Comment:
Cool idea for a small component! I think it would be even better if you mention that it is a button in the name :)

Also, if it always does the same, why would it have different text? I think a nice icon would round this thing up to perfection :)